docs/index.md

```diff
@@ -1,4 +1,6 @@
 Welcome to MasterMind's documentation!
-======================================
+==========================================
 
-TODO
+MasterMind is a management platform to help Fiware Lab users manage and deploy/register IoT Services, such as Context Brokers, IoT Agents, IoT Managers and so forth. It allows users to create individual projects with defined permissions per user, register Docker Swarm clusters to them in order to deploy or register IoT services, as well as registering and managing NSGI subscriptions.
```

Comment:
MasterMind is a management platform for FIWARE Data and IoT services, such as Context Brokers, IoT Agents, IoT Managers and so forth. It allows users to create individual projects with defined permissions per user, register Docker Swarm clusters to them in order to deploy or register IoT services, as well as registering and managing NSGI subscriptions.